src/features/fund/components/fund-account-form.tsx

```diff
@@ -0,0 +1,74 @@
+import { Form } from '@/features/forms/components/form'
+import { useCallback } from 'react'
+import { SubmitButton } from '@/features/forms/components/submit-button'
+import { FormActions } from '@/features/forms/components/form-actions'
+import { z } from 'zod'
+import { toast } from 'react-toastify'
+import { ZERO_ADDRESS } from '@/features/common/constants'
+import { zfd } from 'zod-form-data'
+import { algos } from '@algorandfoundation/algokit-utils'
+import { AlgoAmount } from '@algorandfoundation/algokit-utils/types/amount'
+import { Address } from '@/features/accounts/data/types'
+import SvgAlgorand from '@/features/common/components/icons/algorand'
+import { invariant } from '@/utils/invariant'
+
+const receiverAddressLabel = 'Receiver Address'
+const amountLabel = 'Amount'
+
+const fundFormSchema = zfd.formData({
+  receiver: zfd.text().optional(),
+  amount: zfd.numeric(z.number({ required_error: 'Required', invalid_type_error: 'Required' }).min(0)),
+})
+
+type Props = {
+  onCreateReceiver?: () => Promise<Address>
+  onFund: (receiver: Address, amount: AlgoAmount) => Promise<void>
```

Comment:
maybe we should rename `onFund` to `onFunded`

Reply:
I've changed to onSubmit, which I think is clearer.